locales/en/allocations.json

```diff
@@ -1,7 +1,8 @@
 {
   "complete": "complete",
   "incomplete": "incomplete",
-  "total": "total",
+  "total": "total allocations",
+  "date": "Date of travel",
```

Comment:
I'm sure this content already exists. Would be good to make use  of existing content instead of adding more to maintain.